src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -30,6 +30,7 @@
 
     private static final String REUSED_TAG = "//@reused";
     private static final int AUTHOR_NAME_OFFSET = "author ".length();
+    private static final int AUTHOR_EMAIL_OFFSET = "author-mail ".length();
```

Comment:
```author-mail``` -> ```author-email```

Reply:
This is based on the git syntax and cannot be changed.